packages/devextreme/js/__internal/scheduler/scheduler_options_base_widget.ts

```diff
@@ -0,0 +1,116 @@
+import Widget from '@js/ui/widget/ui.widget';
+import { extend } from '@ts/core/utils/m_extend';
+
+import {
+  DEFAULT_SCHEDULER_INTEGRATION_OPTIONS,
+  DEFAULT_SCHEDULER_INTERNAL_OPTIONS,
+  DEFAULT_SCHEDULER_OPTIONS,
+  DEFAULT_SCHEDULER_OPTIONS_RULES,
+  DEPRECATED_SCHEDULER_OPTIONS,
+} from './utils/options/constants';
+import type {
+  NormalizedView, SafeSchedulerOptions, SchedulerOptionsRule, View,
+} from './utils/options/types';
+import { getCurrentView, getViewOption, getViews } from './utils/options/utils';
+import { SchedulerOptionsValidator, SchedulerOptionsValidatorErrorsHandler } from './utils/options_validator/index';
+
+export class SchedulerOptionsBaseWidget extends Widget<SafeSchedulerOptions> {
+  protected views: NormalizedView[] = [];
+
+  public currentView!: NormalizedView;
+
+  private _optionsValidator!: SchedulerOptionsValidator;
+
+  private _optionsValidatorErrorHandler!: SchedulerOptionsValidatorErrorsHandler;
```

Comment:
We decided not to use the ```_```
So, let remove this symbol

```suggestion
  private optionsValidator!: SchedulerOptionsValidator;

  private optionsValidatorErrorHandler!: SchedulerOptionsValidatorErrorsHandler;
```